### ISSUE TITLE
Running MPV from terminal requests microphone permission

### ISSUE THREAD

**Issue by mhmdanas** [mpv-player/mpv#10874]
### Important Information

- mpv version: 0.35.0
- macOS Version: 13.0.1
- Source of the mpv binary or bundle: Homebrew formula
- If known which version of mpv introduced the problem: unknown

I believe this issue is due to a change in macOS Ventura, but I'm not certain. I do not recall ever having this issue before having upgraded to Ventura.

### Reproduction steps

I've reproduced this bug with Alacritty, iTerm 2, Kitty, and Terminal.app, but the steps will refer to Alacritty, which is my terminal of choice.

1. Open Alacritty
2. Run `tccutil reset Microphone org.alacritty` to reset the microphone permission for Alacritty
3. Quit and then relaunch Alacritty
4. Play any video or audio file, e.g. `mpv --no-config file.mp3`

### Expected behavior
MPV should play the file without asking for microphone access.

### Actual behavior

The terminal MPV is running in asks for microphone permission:

![Microphone access request dialog](https://user-images.githubusercontent.com/32234660/202039072-4722897f-4971-437d-80a6-361d01e6589a.png)

Whether I press "Don't Allow" or "OK," MPV proceeds to play the file, but only after I press either button.

### Log file

https://0x0.st/oIS-.txt

The last line in the logs printed before the dialog above is shown is line 437 ("requested format: 44100 Hz, stereo channels, floatp").

### Additional context

I initially reported this issue on IRC, and I was told this:

```
avih> if you can build mpv yourself, or get it from some other source (e.g. bundled with iiana), it might be worth trying too for reference
```

So I went ahead and installed IINA. I cannot reproduce the issue by running IINA from the app bundle directly. Interestingly however, following the reproduction steps I outlined above but running this command instead of the MPV command does reproduce the same issue:
```
$ # Replace `/Applications` with the location where IINA is installed on your machine
$ /Applications/IINA.app/Contents/MacOS/iina-cli
$ # Or if you installed IINA as a Homebrew cask, this also works:
$ iina
```

The exact same dialog is shown (yes, with Alacritty asking for microphone access, not IINA), and IINA will not launch until any of the two buttons on the permission dialog is pressed.

**Comment by Akemi** [comment]
i can't reproduce this with any of the terminals on the newest macOS and mpv master.

if this problem still persists please open a new issue.
